test/media/media_source_engine_integration.js

```diff
@@ -454,6 +454,19 @@ describe('MediaSourceEngine', () => {
     expect(onMetadata).toHaveBeenCalled();
   });
 
+  it('extracts ID3 metadata from TS when transmuxing', async () => {
```

Comment:
This test is not exactly related to this PR, but it made sense to add it so that this does not happen in the future